chsdi/templates/htmlpopup/personenverkehr_strasse.mako

```diff
@@ -0,0 +1,33 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+<% c['stable_id'] = False %>
+    <tr><td class="cell-left">${_('ch.are.belastung-personenverkehr-strasse.nr')}</td>          <td>${c['attributes']['nr']}</td></tr>
+    <tr><td class="cell-left">${_('ch.are.belastung-personenverkehr-strasse.dwv_fzg')}</td>     <td>${c['attributes']['dwv_fzg']}</td></tr>
+</%def>
+
+<%def name="extended_info(c, lang)">
+<table class="table-with-border kernkraftwerke-extended">
+ <tr><th class="cell-left">${_('ch.are.belastung-personenverkehr-strasse.nr')}</th><td>${c['attributes']['nr'] or '-'}</td></tr>
```

Comment:
I would use <th /> in the first row of a table and not in the first column

Reply:
sorry, i dont understand: